src/thunderbird_accounts/settings.py

```diff
@@ -26,8 +26,10 @@
 IS_TEST = 'test' in sys.argv
 
 if DEBUG:
-    load_dotenv()
-
+    if 'test' in sys.argv:
```

Comment:
```suggestion
    if IS_TEST:
```

It's defined just above there. 😄